src/Eagle.ts

```diff
@@ -82,9 +83,14 @@ export class Eagle {
 
     static dragStartX : number;
     static adjustingLeftWindow : boolean; // true if adjusting left window, false if adjusting right window
+    static adjustingRightWindow : boolean; // true if adjusting left window, false if adjusting right window
 
     static selectedNodeKey : number;
 
+    static nodeDropped : Element;
+    static nodeDropLocation = {x:0, y:0};
+    static pos = {x:0, y:0};
```

Comment:
Does pos have to be a static attribute on the Eagle class? Is it only used inside addNodeToLogicalGraph()?